tests/acceptance/TestHelpers/WebDavHelper.php

```diff
@@ -702,27 +738,35 @@ public static function makeDavRequest(
 		}
 
 		// get space id if testing with spaces dav
-		if ($spaceId === null && $davPathVersionToUse === self::DAV_VERSION_SPACES) {
+		if ($spaceId === null && $davPathVersionToUse === self::DAV_VERSION_SPACES && !\in_array($type, ["public-files", "versions"])) {
 			$path = \ltrim($path, "/");
 			if (\str_starts_with($path, "Shares/")) {
 				$spaceId = self::getSharesSpaceIdForUser(
 					$baseUrl,
-					$doDavRequestAsUser ?? $user,
+					$user,
 					$password,
 					$xRequestId
 				);
 				$path = "/" . preg_replace("/^Shares\//", "", $path);
 			} else {
 				$spaceId = self::getPersonalSpaceIdForUserOrFakeIfNotFound(
 					$baseUrl,
-					$doDavRequestAsUser ?? $user,
+					$user,
 					$password,
 					$xRequestId
 				);
 			}
 		}
 
-		$davPath = self::getDavPath($doDavRequestAsUser ?? $user, $davPathVersionToUse, $type, $spaceId);
+		$uniquePath = $user;
```

Comment:
```suggestion
		$uniquePath = $user;
```
$uniquePath name found confusing to me

Reply:
yes it is. it can represent `username`, `file-id`, `token`, `spcae-id`. Maybe `infixPath` or `suffixPath` will be good here.

Reply:
I will go with `suffixPath`